test/unit/merge.js

```diff
@@ -2415,11 +2415,12 @@ describe('Merge', function() {
           _.omit(movedSrcFile, ['_rev']),
           _.defaults(
             {
+              _id: dstFile._id,
+              path: dstFile.path,
               sides: increasedSides(srcFile.sides, this.side, 1),
               moveFrom: movedSrcFile
             },
-            _.pick(srcFile, ['remote']),
-            _.omit(dstFile, ['_rev'])
+            _.omit(srcFile, ['_rev'])
```

Comment:
I don't understand how the changes in these files relate to the other changes in the PR?

Reply:
It rather seems like a simplification of the expectation and probably doesn't relate to the rest of the PR.
I'll try to move it to another PR.

Reply:
imo a separate commit is fine :)

Reply:
Actually, this change is required. 

Before, we were expecting the final file record to keep all the attributes of the overwritten file (we're testing a move overwriting existing documents at the destination) besides the `remote` attribute (which holds the id and revision of a the associated remote document) which should be the one of the moved file.

This is not true anymore because we're not keeping the full precision of the modification time for the `updated_at` attribute (instead of removing the milliseconds) and this date is not the same for the overwritten file and the overwriting one.
The test was passing before only because the records were created within the same second so they had the same `updated_at` value.